components/studentPortal/layout/SidebarItem.tsx

```diff
@@ -25,25 +26,33 @@ const SidebarItem = ({
 
   return (
     <>
-      <Link onClick={onClick} href={href}>
-        <div
-          className={`flex flex-wrap items-center p-4 cursor-pointer hover:border-l-4 ${
-            activePage === page ? "border-charmander text-charmander" : ""
-          } hover:border-charmander hover:text-charmander ${
-            isDisabled ? "border-gray-500 text-gray-500" : ""
-          }`}
-        >
-          <div>
-            {notifications ? (
-              <div className="relative left-6 top-1 ">
-                <div className="flex w-2 h-2 bg-red-500 rounded-full"></div>
-              </div>
-            ) : null}
-            {icon}
+      {isDisabled ? (
```

Comment:
what was isDisabled being used for before? Are there any use cases where we'd want to disable a sidebar item but not sure the freemium tooltip?

I'm not sure that freemium sidebar item should live inside SidebarItem. maybe we should move this up one layer into the sidebar component? 

That way freemiumSidebarItem and SidebarItem don't know about each other and it is up to the Sidebar to organize them and determine which to show.

Reply:
The isDisabled was created for the freemium users -- but your point is well-taken.  
I have moved the FreemiumSidebarItem to be conditionally rendered based on userRole === "freemium" || userRole === "paid" and moved it to the Sidebar, as it has now been refactored.